### DISCUSSION TITLE
Trash to neo tree

### DISCUSSION THREAD

**Discussion by juan-gomez-hub** [nvim-neo-tree/neo-tree.nvim#1518] [Ideas]
Hello, I share the code of my configuration to use trash-cli, it is quite functional for me and I would like it to be taken into account
`["Y"] = function(state)
            local node = state.tree:get_node()
            local filepath = node:get_id()
            local comman = require("neo-tree.sources.filesystem.commands")
            local filename = node.name
            local ip = require("neo-tree.ui.inputs")
            local function back(result)
              if result then
                if node and filepath then
                  local path = filepath
                  vim.fn.system({ "trash", path })
                  print("successfully deleted" .. path)
                end
              else
                print("deletion was canceled")
                comman.refresh(state)
                return
              end
              comman.refresh(state)
            end
            ip.confirm("Do you want to send the file to the trash?", back)

> `

**Comment by miversen33** [comment]
Feel free to submit this to the wiki. It appears we don't have it in there currently but there have been more than a couple requests like this.

#1037 #202 #860
